cmake/mkldnn.patch

```diff
@@ -28,3 +28,16 @@ index f10feb20..05f47961 100644
  set_property(TARGET ${LIB_NAME} PROPERTY PUBLIC_HEADER ${HEADERS})
  
  target_include_directories(${LIB_NAME} PUBLIC
+diff --git a/src/cpu/jit_avx512_common_conv_kernel.cpp b/src/cpu/jit_avx512_common_conv_kernel.cpp
+index 1bb98fa43..b8b54401f 100644
+--- a/src/cpu/jit_avx512_common_conv_kernel.cpp
++++ b/src/cpu/jit_avx512_common_conv_kernel.cpp
+@@ -3055,7 +3055,7 @@ void jit_avx512_common_conv_bwd_weights_kernel_f32::bias_kernel_3d() {
+ void jit_avx512_common_conv_bwd_weights_kernel_f32
+     ::compute_oh_loop_common()
+ {
+-    assert(jcp.harness == harness_mb_reduction);
```

Comment:
throw an ngraph_errror instead of asserting.

Reply:
this is mkldnn code.